src/isPalindrome.js

```diff
@@ -0,0 +1,13 @@
+export default function isPalindrome(inputString){
+
+  inputString = inputString.toLowerCase();
```

Comment:
Avoid changing the parameter that is provided to a function - this may have unintended consequences outside of the function (known as _side effects_), if the caller did not anticipate that the function would change the provided parameter.  To avoid this possibility, declare a variable within the context of this function to perform the necessary processing:
```js
const array = input.toLowerCase().replace( /[^\w]|_/g, "" ).split( '' )
const reverse = array.reverse().join( '' )
```